onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2786,7 +2832,7 @@ ORT_API(const char*, OrtApis::GetVersionString) {
   return ORT_VERSION;
 }
 
-ORT_API(const char*, OrtApis::GetBuildInfoString) {
+const char* ORT_API_CALL OrtApis::GetBuildInfoString() noexcept {
```

Comment:
why this change? doesn't ORT_API already include `ORT_API_CALL` and `NO_EXCEPTION`?

https://github.com/microsoft/onnxruntime/blob/7880342e5e50159f59b9f9c0bd0a74ecc1542549/include/onnxruntime/core/session/onnxruntime_c_api.h#L143